modules/gitops-enabling-the-creation-of-aggregated-cluster-roles.adoc

```diff
@@ -0,0 +1,270 @@
+// Module included in the following assembly:
+//
+// * declarative_clusterconfig/customizing-permissions-by-creating-aggregated-cluster-roles.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-enabling-the-creation-of-aggregated-cluster-roles_{context}"]
+= Enabling the creation of aggregated cluster roles
+
+To enable the creation of aggregated cluster roles for the Argo CD Application Controller component of a cluster-scoped Argo CD instance, you must configure the corresponding field by editing the YAML file of the Argo CD custom resource (CR).
```

Comment:
Suggestion: Can we add the explicit name of the corresponding field in this sentence? As it is the first sentence of the procedure, I feel that providing the name of the corresponding field improves the existing sentence.